src/main/java/core/basesyntax/service/impl/operation/ReturnOperation.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl.operation;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.model.Fruit;
+
+public class ReturnOperation implements OperationHandler {
```

Comment:
The class `ReturnOperation` should be in a package that reflects its purpose, following the checklist suggestion to keep all logic not in a single package. Consider placing this class in a `strategy` package as it's a part of the Strategy pattern implementation.